pkg/simple/client/k8s/options.go

```diff
@@ -44,12 +48,26 @@ type KubernetesOptions struct {
 }
 
 // NewKubernetesOptions returns a `zero` instance
-func NewKubernetesOptions() *KubernetesOptions {
-	return &KubernetesOptions{
-		KubeConfig: "",
-		QPS:        1e6,
-		Burst:      1e6,
+func NewKubernetesOptions() (option *KubernetesOptions) {
+	option = &KubernetesOptions{
+		QPS:   1e6,
+		Burst: 1e6,
+	}
+
+	// make it be easier for those who wants to run api-server locally
+	homePath := homedir.HomeDir()
+	if homePath == "" {
+		// try os/user.HomeDir when $HOME is unset.
+		if u, err := user.Current(); err == nil {
+			homePath = u.HomeDir
+		}
+	}
+
+	userHomeConfig := path.Join(homePath, ".kube/config")
+	if _, err := os.Stat(userHomeConfig); err == nil {
+		option.KubeConfig = userHomeConfig
 	}
+	return
```

Comment:
IMO, the `return` line is unnecessary for here.

Reply:
![image](https://user-images.githubusercontent.com/1450685/122513725-5e621200-d03d-11eb-9d1d-47ccf52a956c.png)

Reply:
I have no problem as https://github.com/golang/go/issues/12168

Reply:
> I have no problem as [golang/go#12168](https://github.com/golang/go/issues/12168)

I didn't get your point. Are you trying to say that there's no error in your IDE?

Reply:
I mean that I've seen the [declined proposal](https://github.com/golang/go/issues/12168) about return statement at the very end of a function, so I have no problem of this return statement.